recommendation/templates/recommendation/recommendation_form.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'dashboard/index.html' %}
+{% block recommend_form %}
+
+{% load static %}
```

Comment:
Please refer to the comment above for getting rid of these import statements in this file and having them only in the index.html (recommendation)

Reply:
must be loaded here or else the look changes

Reply:
okay, resolving this now